src/rendering/SoRenderManagerP.cpp

```diff
@@ -133,7 +134,12 @@ SoRenderManagerP::setClippingPlanes(void)
   float nearval = -box.getMax()[2];
   float farval = -box.getMin()[2];
 
-  if (farval <= 0.0f) return;
+  if (!camera->isOfType(SoOrthographicCamera::getClassTypeId()) && farval <= 0.0f) return;
+
+  if (box.isEmpty()) {
+    nearval = 0;
```

Comment:
If the box is empty I would set the default value for the nearval which is 1.0

Reply:
Thanks for the PR, I run a simple test and it did not show any unwanted effects. After resolving this conversation I am happy to merge the PR.

Reply:
Thanks, I updated the nearval to 1